frame/support/test/tests/pallet_ui/storage_ensure_span_are_ok_on_wrong_gen_unnamed.stderr

```diff
@@ -55,76 +55,80 @@ error[E0277]: the trait bound `Bar: WrapperTypeEncode` is not satisfied
-  --> tests/pallet_ui/storage_ensure_span_are_ok_on_wrong_gen_unnamed.rs:21:12
-   |
-21 |     #[pallet::storage]
-   |               ^^^^^^^ the trait `EncodeLike` is not implemented for `Bar`
-   |
-   = help: the following other types implement trait `EncodeLike<T>`:
-             <&&T as EncodeLike<T>>
-             <&T as EncodeLike<T>>
-             <&T as EncodeLike>
-             <&[(K, V)] as EncodeLike<BTreeMap<LikeK, LikeV>>>
-             <&[(T,)] as EncodeLike<BTreeSet<LikeT>>>
-             <&[(T,)] as EncodeLike<BinaryHeap<LikeT>>>
-             <&[(T,)] as EncodeLike<LinkedList<LikeT>>>
-             <&[T] as EncodeLike<Vec<U>>>
-           and $N others
-   = note: required for `Bar` to implement `FullEncode`
-   = note: required for `Bar` to implement `FullCodec`
-   = note: required for `frame_support::pallet_prelude::StorageValue<_GeneratedPrefixForStorageFoo<T>, Bar>` to implement `StorageEntryMetadataBuilder`
+ --> tests/pallet_ui/storage_ensure_span_are_ok_on_wrong_gen_unnamed.rs:1:1
+  |
+1 | #[frame_support::pallet]
+  | ^^^^^^^^^^^^^^^^^^^^^^^^ the trait `EncodeLike` is not implemented for `Bar`
+  |
+  = help: the following other types implement trait `EncodeLike<T>`:
+            <&&T as EncodeLike<T>>
+            <&T as EncodeLike<T>>
+            <&T as EncodeLike>
+            <&[(K, V)] as EncodeLike<BTreeMap<LikeK, LikeV>>>
+            <&[(T,)] as EncodeLike<BTreeSet<LikeT>>>
+            <&[(T,)] as EncodeLike<BinaryHeap<LikeT>>>
+            <&[(T,)] as EncodeLike<LinkedList<LikeT>>>
+            <&[T] as EncodeLike<Vec<U>>>
+          and $N others
+  = note: required for `Bar` to implement `FullEncode`
+  = note: required for `Bar` to implement `FullCodec`
+  = note: required for `frame_support::pallet_prelude::StorageValue<_GeneratedPrefixForStorageFoo<T>, Bar>` to implement `StorageEntryMetadataBuilder`
+  = note: this error originates in the attribute macro `frame_support::pallet` (in Nightly builds, run with -Z macro-backtrace for more info)
 
 error[E0277]: the trait bound `Bar: WrapperTypeEncode` is not satisfied
-  --> tests/pallet_ui/storage_ensure_span_are_ok_on_wrong_gen_unnamed.rs:21:12
-   |
-21 |     #[pallet::storage]
-   |               ^^^^^^^ the trait `WrapperTypeEncode` is not implemented for `Bar`
-   |
-   = help: the following other types implement trait `WrapperTypeEncode`:
-             &T
-             &mut T
-             Arc<T>
-             Box<T>
-             Cow<'a, T>
-             Rc<T>
-             Vec<T>
-             bytes::bytes::Bytes
-           and $N others
-   = note: required for `Bar` to implement `Encode`
-   = note: required for `Bar` to implement `FullEncode`
-   = note: required for `Bar` to implement `FullCodec`
-   = note: required for `frame_support::pallet_prelude::StorageValue<_GeneratedPrefixForStorageFoo<T>, Bar>` to implement `StorageEntryMetadataBuilder`
+ --> tests/pallet_ui/storage_ensure_span_are_ok_on_wrong_gen_unnamed.rs:1:1
```

Comment:
Looks like the old line numbers were a lot more informative. I would revert whatever changed this